aspnetcore/tutorials/build-your-first-blazor-app/samples_snapshot/3.x/Counter2.razor

```diff
@@ -4,9 +4,9 @@
 
 <p>Current count: @currentCount</p>
 
-<button class="btn btn-primary" onclick="@IncrementCount">Click me</button>
+<button class="btn btn-primary" @onclick="@IncrementCount">Click me</button>
```

Comment:
Should we log a follow up to remove the extra `@` in preview 7? @guardrex the fact that it requires `@IncrementCount` instead of `IncrementCount` is a bug we're fixing in preview 7.

Reply:
Created #12766 for Pre7 directive updates. Thus far, it includes this and `@(...)` from above.